### ISSUE TITLE
Login/Reset page: Inline error in displayed in English on French page

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#43]
URL: https://hcdev.openplus.ca/fr/user/login
User Agent: Chrome
Language: French

Issue Description: Inline error message is in English on the French page. 
Steps to reproduce:
From the French Login page (or Reset page) leave page blank and trigger the "Se Connecter" button
Inline error for "nom d'utilisateur" appears but is in English.

![image](https://user-images.githubusercontent.com/65728700/128418158-36b193d0-7b06-4434-815d-02a55545a835.png)
![image](https://user-images.githubusercontent.com/65728700/128418364-4a385ce1-36ce-4bff-8af5-caa28333b125.png)

Expected results:
Error messages should be in French. Should follow WET error handling. avoid inline error messages.

**Comment by davidgotlieb** [comment]
This is related to tooltips, which have been disabled.

**Comment by qanitah** [comment]
These still present on both pages.
![image](https://user-images.githubusercontent.com/14280062/136273319-ea7e0052-0e72-4700-bb93-06ca2745fb99.png)

**Comment by ndolter** [comment]
I also found an additional example on the Dashboard page:
![image](https://user-images.githubusercontent.com/68117281/136394630-ef2ad1a5-4843-452e-b9a8-837e10ac502f.png)

**Comment by echouin** [comment]
@mmiron-HC from OP: tooltips was removed. This is now resolved.

**Comment by ndolter** [comment]
Validated that the tooltip messages have been removed.
